lib/routers/establishment.js

```diff
@@ -16,7 +17,7 @@ router.get('/', (req, res, next) => {
     .catch(next);
 });
 
-router.use('/:id', (req, res, next) => {
+router.use('/:id', permissions('establishment.read', { establishment: 'id' }), (req, res, next) => {
```

Comment:
There's no second argument to the permissions middleware that I can see, it just uses `req.params`. So I think this needs to be `router.use('/:establishment', ...)` and `req.params.establishment` lower down.

Reply:
yep - right you are! Obvs left over from the previous implementation

Reply:
_however_ in asl-service the param used is still `:id` so it would need to be changed there and bumped before it can be changed here

Reply:
In which case, what I should have done was not have merged your other PR, and left this as it is.

Whoops.